noetic/distribution.yaml

```diff
@@ -2160,6 +2160,12 @@ repositories:
       url: https://github.com/ros-visualization/gl_dependency.git
       version: kinetic-devel
     status: maintained
+  gpp:
```

Comment:
I have to say that the name of this package is a little too short for my liking.  In the wide world of robotics, it would be hard to distinguish exactly what this package did.  I'd much prefer "global_planner_pipieline", but just renaming the key here wouldn't be enough.  The name of the package in the source repository would also have to be updated.

Reply:
agreed - the name is a little too ambiguous. however, it's not associated with any package: the repository has no meta-package and the package-names are gpp_plugin and gpp_interface - which i hope is less easy to mixup. I could nonetheless rename the repository to global_planner_pipeline. I fear that doing so might be more confusing for users looking for sources of their gpp_<bla> packages. What do you think?

Reply:
any verdict?

Reply:
> and the package-names are gpp_plugin and gpp_interface

I agree that the repo name is "too short", but considering that the package names are longer it seems to be okey to me.

Reply:
@clalancette friendly ping

Reply:
Yeah, good point.  It's OK with me, feel free to proceed with this PR.